cdk/kraken/src/deploy.ts

```diff
@@ -60,21 +74,14 @@ export class DeployJob extends CheckoutJob {
           },
         },
         {
-          name: 'Deploy',
-          run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
-
-          # get repo name from synth step
-          RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}
```

Comment:
`RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}` is necessary to use the `$RELEASE_NAME` env variable in the `git commit -m "chore(k8s): deploy $RELEASE_NAME"` line you added

Reply:
Wups, thanks